pkg/deploy/registry/deploylog/rest.go

```diff
@@ -173,3 +178,26 @@ func (g *podGetter) Get(ctx kapi.Context, name string) (runtime.Object, error) {
 	}
 	return g.podsNamespacer.Pods(namespace).Get(name)
 }
+
+// returnApplicationPodName tries to resolve the name for the oldest pod for the target deployment.
+func (r *REST) returnApplicationPodName(target *kapi.ReplicationController) (string, error) {
```

Comment:
When we add the support proposed by Eric upstream, would the client side be responsible for doing the fan out?  I.e. would we have a Reaper equivalent (LogMulitplexer) for a DC and Deployment?  I assume it's not right to do it on the server, but makes it harder.